File: backend/src/main/java/com/project/dto/assessment/response/AssessmentResultDTO.java
package com.project.dto.assessment.response;

import java.util.List;

import lombok.Data;

@Data
public class AssessmentResultDTO {

    private Double averageScore;
    private String resultTitle;
    private String suggestion;
    private String disclaimer =
        "This assessment is not a medical diagnosis.";
}

File: backend_dotnet/SafeMind.Core/DTOs/Assessment/Response/AssessmentResultDTO.cs
namespace SafeMind.Core.DTOs.Assessment.Response
{
    public class AssessmentResultDTO
    {
        public double AverageScore { get; set; }
        public string ResultTitle { get; set; } = string.Empty;
        public string Suggestion { get; set; } = string.Empty;
        public string Disclaimer { get; set; } = "This assessment is not a medical diagnosis.";
    }
}
